### PR TITLE
arecord: add example list audio devices and intractive interface

### PR DESCRIPTION
**Pull request by Ray6464**
- [] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ x The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/arecord.md [hunk 1]
```diff
@@ -14,3 +14,11 @@
 - Record a snippet and save it as mp3 (finish with Ctrl-C when done):
 
 `arecord -vv --format=cd --file-type raw | lame -r - {{path/to/file.mp3}}`
+
+- List all sound cards and digital audio devices
```
Comment (gutjuri) [inline pages/linux/arecord.md:18]:
```suggestion
- List all sound cards and digital audio devices:
```

Apparently this is needed for formatting

File: pages/linux/arecord.md [hunk 2]
```diff
@@ -14,3 +14,11 @@
...
+
+`arecord -l`
+
+- Allow intractive interface like play/pause via spacebar/enter
```
Comment (gutjuri) [inline pages/linux/arecord.md:22]:
```suggestion
- Allow intractive interface like play/pause via spacebar/enter:
```
This also

**Review by gutjuri — CHANGES_REQUESTED** [review]

File: pages/linux/arecord.md [hunk 3]
```diff
@@ -14,3 +14,11 @@
...
 
 `arecord -vv --format=cd --file-type raw | lame -r - {{path/to/file.mp3}}`
+
+- List all sound cards and digital audio devices:
+
+`arecord -l`
+
+- Allow intractive interface like play/pause via spacebar/enter:
+
+`arecord -i`
```
Comment (navarroaxel) [inline pages/linux/arecord.md:24]:
```suggestion
`arecord --interactive`
```

File: pages/linux/arecord.md [hunk 4]
```diff
@@ -14,3 +14,11 @@
 - Record a snippet and save it as mp3 (finish with Ctrl-C when done):
 
 `arecord -vv --format=cd --file-type raw | lame -r - {{path/to/file.mp3}}`
+
+- List all sound cards and digital audio devices:
+
+`arecord -l`
```
Comment (navarroaxel) [inline pages/linux/arecord.md:20]: Please use long argument format:
```suggestion
`arecord --list-devices`
```

File: pages/linux/arecord.md [hunk 5]
```diff
@@ -14,3 +14,11 @@
...
+
+- Allow intractive interface like play/pause via spacebar/enter:
```
Comment (navarroaxel) [inline pages/linux/arecord.md:22]:
```suggestion
- Allow interactive interface (e.g. use space-bar or enter to play or pause):
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @Ray6464, just a few comments.

**Review by navarroaxel — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @Ray6464!

**Pull request merged**